_episodes/21-carpentries.md

```diff
@@ -118,9 +140,9 @@ under a permissive [open license]({{ page.root }}/LICENSE).
 You may use them whenever and however you want,
 provided you cite the original source.
 
-### Using the Names
+### Using the Names and Logos
 
-The names "Software Carpentry" and "Data Carpentry"
+The names "Software Carpentry", "Data Carpentry" and "Library Carpentry"
 and their respective logos
 are all trademarked.
 You may only call a workshop a Software Carpentry or Data Carpentry workshop if:
```

Comment:
Likely this should read "Software Carpentry, Data Carpentry, or Library Carpentry workshop if:"

Reply:
good catch!